.github/workflows/ci.yml

```
@@ -52,8 +52,7 @@ jobs:
 
       - name: Run tests
         run: |
```

On peut le mettre sur la même ligne, si on veut harmoniser avec les autres `run: …`.

```suggestion
        run: bin/rails test:all
```